lib/house.rb

```diff
@@ -0,0 +1,32 @@
+class House
+  attr_reader :price, :address, :rooms, :house_area
```

Comment:
Keep an eye on the interaction pattern, we were really looking for a `price` method that would return the price of the house as an integer - this would also help with a few methods below